src/app/games/lodestone/LodestoneTutorial.ts

```diff
@@ -30,32 +31,32 @@ export class LodestoneTutorial {
             LodestoneState.getInitialState(),
             ['#lodestone_push_diagonal'],
-            `Congratulations!`,
-            `This is not the right lodestone, try again.`,
+            $localize`Congratulations!`,
+            $localize`This is not the right lodestone, try again.`,
         ),
         TutorialStep.informational(
-            $localize`The pushing lodestone`,
-            $localize`Upon placing a lodestone on the board, it will move all pieces it acts upon (according to the lodestone side and direction) simultaneously. Let us first see how the push lodestone acts on the pieces. All the opponent's pieces aligned with the lodestone, as indicated by the triangles' orientations, will be pushed one square away from the lodestone. An opponent's piece will be blocked in case it encounters on its way either one of your piece, a lodestone, or another blocked piece. Finally, if an opponent's piece falls out of the board, it is considered as captured.`,
+            $localize`The repelling lodestone`,
+            $localize`Upon placing a lodestone on the board, it will move all pieces it acts upon (according to its direction and orientation) simultaneously. Let us first see how the repelling lodestone acts on the pieces. All the opponent's pieces aligned with the lodestone, as indicated by the triangles' orientations, will be pushed one square away from the lodestone. An opponent's piece will be blocked in case it encounters on its way either one of your piece, a lodestone, or another blocked piece. Finally, if an opponent's piece falls out of the board, it captured.`,
             LodestoneState.getInitialState(),
         ),
         TutorialStep.informational(
-            $localize`The pulling lodestone`,
-            $localize`When the lodestone is on its pull side it will pull your pieces one square towards it. In case your pieces encounter a lodestone on their way or another blocked piece, they are blocked. However, if they encounter an opponent's piece, they will crush it and the opponent's piece is considered as captured.`,
+            $localize`The attracting lodestone`,
+            $localize`When the lodestone is on its attracting side it will pull your pieces one square towards it. In case one of your pieces encounters a lodestone on its way or another blocked piece, it is blocked. However, if it encounter an opponent's piece, it will capture it.`,
```

Comment:
it will attract :p ? or too much redundancy :D ?

Reply:
oui me semble que c'est plus clair de dire « une pierre attractive tire les pièces » plutôt que « une pierre attractive attire les pièce » non ? (éviter une définition tautologique !)